src/PatrolFilter/index.js

```diff
@@ -48,13 +51,38 @@ const PatrolFilter = ({ className }) => {
     patrolFilterTracker.track('Change Search Text Filter');
   }, []);
 
-  const resetSearch = useCallback((e) => {
-    e.stopPropagation();
+  const resetSearch = useCallback((event = null) => {
+    event?.stopPropagation();
     setFilterText('');
 
     patrolFilterTracker.track('Clear Search Text Filter');
   }, []);
 
+  const resetDateRange = useCallback(() => {
```

Comment:
Why do we define these methods `resetDateRange` and `resetFiltersPopover` if they are not reused and add memoization processing? We could simply write the code they have directly in `resetAllFilters` right?